app/models/invenio_rdm_record_converter.rb

```diff
@@ -441,6 +442,21 @@ def normalize_date(date_string)
   end
 
   def funding(file_id)
-    @@funding_data[file_id] || [{}]
+    funding_sources = @@funding_data[file_id]
+
+    if funding_sources.blank?
+      return []
+    end
+
+    funding_sources.map do |source|
+      source == BLANK_FUNDER_SOURCE
+
+      # if the title is blank, replace it with the award number
+      if source[:award][:title].blank?
+        source[:award][:title] = source[:award][:number]
+      end
```

Comment:
I guess we'll have to see if there are funding entries that are missing other required fields when we import. (`funder.name`, `award.number`). If we do we will have to revisit.

Reply:
I was looking at funding data from production and I think everything else is there, but we can run a script to make sure.